src/Symfony/Component/Security/Core/Authentication/RememberMe/PersistentToken.php

```diff
@@ -39,7 +38,15 @@ public function __construct(string $class, string $userIdentifier, string $serie
             throw new \InvalidArgumentException('$tokenValue must not be empty.');
         }
 
-        $this->class = $class;
+        if ($tokenValue instanceof \DateTime) {
```

Comment:
this condition looks wrong to me. When using the legacy signature, `$tokenValue` will contain the value of `$series`, not of `$lastUsed`